src/v/kafka/server/group_recovery_consumer.cc

```diff
@@ -90,14 +91,19 @@ group_recovery_consumer::operator()(model::record_batch batch) {
                      .cmd;
 
         auto [group_it, _] = _state.groups.try_emplace(val.group_id);
+        vlog(
```

Comment:
Should this commit be swapped with the next one? I think it uses the printing operators defined there